.github/workflows/CI.yml

```diff
@@ -29,6 +30,7 @@ jobs:
           sf plugins install @salesforce/sfdx-scanner
 
       - name: Run SFDX Scanner - Report findings as comments
+        if: github.event_name == 'pull_request'
```

Comment:
Is this necessary? The triggering event is always `pull_request`

Reply:
This is funny - it's a legacy line from when your scanner would fail if it wasn't run on a pull request. Happy to remove it, but because your action doesn't fail gracefully AFAIK, it's here so this can still be run on workflow dispatch, if desired.